docs-src/python/how-to-use-the-python-sdk-patching-api-final-in-python.md

```diff
@@ -0,0 +1,39 @@
+---
+id: how-to-use-the-python-sdk-patching-api-final-in-python
+title: Review the final Python SDK Patching API implementation
+sidebar_label: Final Python SDK patching API implementation
+description: Heres a sample implementation of the final patching in new code using the Python SDK's patching API.
+tags:
+- version
+- python sdk
+- code sample
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/edu-1171/version_your_workflows/workflow_4_patch_complete_dacx.py. -->
+
+Now, you want to update your code to run `post_patch_activity` instead. This represents your desired end state.
+
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/edu-1171/version_your_workflows/workflow_4_patch_complete_dacx.py">View source code</a>
+
+```python
+from datetime import timedelta
+
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
+    from activities import post_patch_activity
+# ...
+@workflow.defn
+class MyWorkflow:
+    @workflow.run
+    async def run(self) -> None:
+        self._result = await workflow.execute_activity(
+            post_patch_activity,
+            schedule_to_close_timeout=timedelta(minutes=5),
+        )
+
+    @workflow.query
```

Comment:
Is this code (the query) necessary for what is being explained here? If not maybe remove?

Reply:
Removed.